src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/router/TransitRouter.java

```diff
@@ -221,7 +223,20 @@ private Collection<DefaultAccessEgress> getAccessEgresses(
       accessRequest.preferences().street().maxAccessEgressDuration().valueOf(streetRequest.mode())
     );
 
-    var results = new ArrayList<>(accessEgressMapper.mapNearbyStops(nearbyStops, isEgress));
+    var isAccess = !isEgress;
+    List<DefaultAccessEgress> results = new ArrayList<>(
+      accessEgressMapper.mapNearbyStops(nearbyStops, isEgress)
+    );
+    if (streetRequest.mode() == StreetMode.CAR_HAILING) {
```

Comment:
@t2gran What should the extension point be?

Reply:
If you want we can take a look at this together...